manifest.json

```diff
@@ -237,6 +235,37 @@
             }
           }
         ]
+      },
+      "includeTwitterMetaTags": {
+        "oneOf": [
+          {
+            "properties": {
+              "includeTwitterMetaTags": {
+                "enum": [false]
+              }
+            }
+          },
+          {
+            "properties": {
+              "includeTwitterMetaTags": {
+                "enum": [true]
+              },
+              "twitterUsername": {
+                "title": "Twitter username",
+                "description": "The username your business uses on Twitter, for example: VTEXcommerce. (Do not include the @ sign)",
+                "type": "string"
+              },
+              "twitterCard": {
+                "title": "Twitter Card",
+                "description": "Please choose the type for the Twitter Card",
+                "type": "string",
+                "default": "summary",
+                "enum": ["photo", "summary", "summary_large_image"],
+                "enumNames": ["Photo", "Summary", "Summmary Large Image"]
```

Comment:
Same thing here. Make sure to use message IDs for all strings that are shown to users, otherwise these texts will always be in English regardless of the currently active Admin language.